lib/src/overlay_date_time_picker.dart

```diff
@@ -99,7 +103,13 @@ class _OverlayDateTimePickerState extends State<OverlayDateTimePicker> {
     debugLabel: "Overlay Date Time Picker - Button",
   );
 
-  bool _isShown = false;
+  bool _isShown = true;
+
+  @override
+  void initState() {
```

Comment:
no need to use initstate, you can use:

`late bool _isShown = widget.isShown;`